Updates to the roles and expectations of proposals, projects, teams (#545)
cncf/tag-security: governance/process.md

# Proposals, projects, and team

> In addition to the TOC driven activities of the SIG, members may propose

Reviewer: ```suggestion
In addition to the [TOC](https://www.cncf.io/people/technical-oversight-committee/)
driven activities of the group, members may propose
``

    * The scope of the work required

1. **Ask the group for collaboration:** Rather than immediately beginning work

> on a solution, add the issue to a meeting agenda by placing the link and the POC

Reviewer: ```suggestion
on a solution, add the issue to a meeting agenda by placing the link and the 
person who will present the proposal
```

in the ["Planned Meeting" area of the meeting

> notes](https://github.com/cncf/sig-security#meeting-times) and bring the issue up for 

Reviewer: ```suggestion
notes](https://github.com/cncf/tag-security#meeting-times) and bring the issue up for 
```

discussion at the scheduled meeting, explain it, and ask if others are interested
 in collaborating. This ensures that solutions are created with multiple 
perspectives as well as verifies there is community interest in pursuing the
proposal. The outcome of this conversation will be:
    * The group agrees that this problem is appropriate scope

>   Group.

Reviewer: ```suggestion
```

    * Criteria for completion are added to the issue that include a "definition
  of done", ideally with validation by the target audience. Also note in the
  issue if it will be a time-bounded project with a defined deliverable or and 

>   on-going stream.

Reviewer: ```suggestion
  on-going stream of work (the latter would typically be proposed only after at least one, usually multiple, projects have been completed.
```

  another member of the group who can attend the meeting is familiar with
  progress in case questions arise.
    * It's strongly encouraged to include a checklist in the Isssue
  that shows what has been done and what work remains and should include a

> retrospective as appropriate.

Reviewer: what does "as appropriate" mean?  maybe just add to template for now?

Pull Request to indicate their vote for or against. Members will have a week
after the vote begins to leave their vote. Quorum is taken to be 2/3 the number
of companies who have been active in the past month (via issue comment or 
meeting attendance as recorded in the public minutes).

1. **Support the project going forward.** Some projects require sustainment and
   maintenance to ensure continued relevance for the community.  When work is 

> completed, the body of the Pull Request should specify if it's expected to require

Reviewer: do you mean Pull Request description?  I would think there would be a new proposal (issue) with a description of the expectation/plan/idea for on-going work
